### DISCUSSION TITLE
Map requires new error....

### DISCUSSION THREAD

**Discussion by vi5hnuu** [react-native-maps/react-native-maps#4756] [Q&A]
### Summary

I was foolowing a tutorial on udemy and i got this error that map requires new...
i saw the documentation and also read about the same issue on the github but couldn't decipher what's going on.

i tried opening the code snippet on the slack as shown on the expo react native docs of mapview.

### Reproducible sample code

```TSX
import { View, Text, StyleSheet } from 'react-native'
import React from 'react'
import MapView from 'react-native-maps'


const Map = () => {
  return new MapView({
    initialRegion: {
      longitude: 79.0668409,
      latitude: 30.735196,
      latitudeDelta: 0.0922,
      longitudeDelta: 0.0421,
    },
    style: styles.map
  })
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
  },
  map: {
    width: '100%',
    height: '100%',
  },
});
export default Map
```

### Steps to reproduce

just use the MapView class as show on the code it will throw the error at runtime...

### Expected result

google maps on screen 

### Actual result

...

### React Native Maps Version

8.19.2

### What platforms are you seeing the problem on?

Android

### React Native Version

8.19.2

### What version of Expo are you using?

SDK 46

### Device(s)

redmi note 9

**Comment by monholm** [comment]
This doesn't seem to be an issue with `react-native-maps`, so I'm converting this to a discussion :)

Try something like:

```ts
import React from 'react';
import {StyleSheet} from 'react-native';
import MapView from 'react-native-maps';

const Map = () => {
  return (
    <MapView
      initialRegion={{
        longitude: 79.0668409,
        latitude: 30.735196,
        latitudeDelta: 0.0922,
        longitudeDelta: 0.0421,
      }}
      style={styles.map}
    />
  );
};

const styles = StyleSheet.create({
  map: {
    width: '100%',
    height: '100%',
  },
});
export default Map;
```
